docs/whats-new/release-candidate-1.md

```diff
@@ -0,0 +1,777 @@
+
+To start the upgrade process, right-click your AppHost project in Visual Studio and select the option `Upgrade`.
+
+![Upgrade using upgrade assistant](images/upgrade-assistant-1.png)
+
+If you don't yet have the Upgrade Assistant installed, then Visual Studio will guide you to the steps of how to install the extension. Once you have it installed, you will see the following option when you select the `Upgrade` option on your AppHost project:
+
+![Upgrade assistant options](images/upgrade-assistant-2.png)
+
+This will then do some analysis in your project and provide you with a report similar to the following detailing what will be updated. All you need to do now is to click the `Upgrade Selection` button to start the upgrade process:
+
+![Upgrade confirmation](images/upgrade-assistant-confirmation.png)
+
+That's it! You have successfully upgraded your project to .NET Aspire 9.0 RC 1 using the Upgrade Assistant.
+
+## Dashboard
+
+### Manage resource lifecycle
+
+The most requested feature for the dashboard is to manage the lifecycles of your orchestrated named resources. Specifically, the ability to stop, start, and restart resources. This feature is now available in .NET Aspire 9.
+
+<!-- markdownlint-disable MD033 -->
+<video controls src="https://github.com/user-attachments/assets/3937ca4e-4ac9-4af7-938d-d016aa6277dc" title="Demo of managing resource lifecycle in the dashboard"></video>
+<!-- markdownlint-enable MD033 -->
+
+This feature works for projects, containers and executables. It enables restarting indidivual resources without having to restart the entire app host. For projects, if the debugger is attached, on restart, the debugger will be re-attached.
+
+- Responsive design
+- Mobile friendly
+
+### Sensitive properties, volumes and health checks in resource details
+
+Resource details has seen a lot of improvements:
+
+- Properties can be marked as sensitive, automatically masking them in the dashboard UI. This is a security feature to avoid accidently disclosing keys or passwords when screen sharing the dashboard with other people. For example, container arguments could pass sensitive information and so are masked by default.
+
+- Configured container volumes are listed in resource details.
+
+- Detailed information about resource health checks, a new feature in Aspire 9, are listed in resource details. This information is useful to understand why a resource might be marked as unhealthy. Find out more about health checks [here](#resource-health-checks).
+
+### Coloreful console log
+
+[ANSI escape codes](https://wikipedia.org/wiki/ANSI_escape_code) are used to add text formatting to text. For example, foreground and background colors, bold, underline, italics, etc. The dashboard console logs page previously supported rendering one ANSI escape code at a time and failed when they were combined together. The console logs page could show red text, but it couldn't show text that is red and bold at the same time.
```

Comment:
I feel like this is easier to understand:

```suggestion
[ANSI escape codes](https://wikipedia.org/wiki/ANSI_escape_code) format text in terminals by controlling colors (foreground and background) and styles like bold, underline and italics. Previously, the dashboard's console logs page could only render one ANSI escape code at a time, failing when multiple codes were combined. For example, it could display red text, but not text that was both red and bold.
```